package.json

```diff
@@ -3,13 +3,21 @@
   "version": "1.9.0",
   "description": "Experimental implementation of a new declarative API for gesture handling in react-native",
   "scripts": {
+    "prepare": "rm -r dist/ ; yarn build",
     "test": "jest",
+    "build": "yarn tsc",
     "precommit": "lint-staged",
     "release": "npm login && release-it",
-    "ts-check": "yarn tsc",
-    "lint-check": "yarn prettier --check src/ examples/Example/src"
+    "ts-check": "yarn tsc --noEmit",
+    "lint-check": "yarn eslint src/ && yarn prettier --check src/"
   },
-  "main": "index.js",
+  "exports": {
+    ".": "./dist/index.js",
+    "./": "./dist/",
+    "./package.json": "./package.json",
+    "./RNGestureHandler.podspec": "./RNGestureHandler.podspec"
```

Comment:
I'm not sure if any other files should be included here, let me know.

I'll test if this is needed after today's changes.

Reply:
Removed in 314bb40.